Woof/Woof/Model/LocalStorage/LocalStorage.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+
+/// A persistent storage designed for managing a collection of values.
+/// These values are stored and retrieved using a key that uniquely identifies the value.
+/// The key is used to find the data within the storage.
+class LocalStorage {
+    /// Instance of persistent storage with the specified database name.
+    let localStorage = UserDefaults(suiteName: "woofLocalStorage")
```

Comment:
Could you please explain why it has internal access level?

Reply:
I fix it to private access level, direct access to the storage from outside is not needed